libraries/AP_Motors/AP_MotorsMulticopter.cpp

```diff
@@ -176,6 +176,13 @@ const AP_Param::GroupInfo AP_MotorsMulticopter::var_info[] = {
 
     // 38 RESERVED for BAT_POW_MAX
     
+    // @Param: BAT_IDX
+    // @DisplayName: Battery compensation index
+    // @Description: Which battery monitor should be used for doing compensation
+    // @Range: 0 1
```

Comment:
Should this be values instead of range? If you want to go with range, then add the `@Increment: 1`.